### PR TITLE
check mod_rewrite, json and fileinfo

### PR DESCRIPTION
**Pull request by mariavilaro**
Add checks for mod_rewrite, json and fileinfo

### CODE REVIEW

File: install.php [hunk 1]
```diff
@@ -124,7 +124,10 @@
                     { code: 'mbstringLibrary', label: 'Mbstring PHP Extension is required' },
                     { code: 'sslLibrary', label: 'OpenSSL PHP Extension is required' },
                     { code: 'zipLibrary', label: 'ZipArchive PHP Library is required' },
-                    { code: 'gdLibrary', label: 'GD PHP Library is required' }
+                    { code: 'gdLibrary', label: 'GD PHP Library is required' },
+                    { code: 'mod_rewrite', label: 'PHP mod_rewrite module is required' },
```
Comment (LukeTowers) [inline install.php]: OctoberCMS doesn't actually require mode_rewrite on non-Apache servers, you'd need to add an additional check to see if this is an Apache server.

Comment (mariavilaro) [reply install.php]: True, I changed that and something more.
And regarding json extension check, and before you say "As of PHP 5.2.0, the JSON extension is bundled and compiled into PHP by default.", I managed to find a PHP 5.6 misconfigured server without JSON enabled...

**Comment by LukeTowers** [comment]
Thanks @mariavilaro! My apologies for taking so long to get back to this, I appreciate the work you put into making October better :)

**Comment by aurukas** [comment]
there's something wrong with the mod_rewrite test. It fails on MAMP Mac OS, although mod_rewrite functions perfectly.

**Comment by LukeTowers** [comment]
@aurukas see https://github.com/octobercms/october/issues/3673 for further discussion

**Pull request merged**